ext/curl/share.c

```diff
@@ -21,6 +21,8 @@
 #endif
 
 #include "php.h"
+#include "ext/standard/php_array.h"
```

Comment:
What function requires pulling in this header?

Reply:
This was a leftover from when I was using a sort function from there, and I forgot to remove it when I changed the implementation. Thanks for catching it!